INSTALL.md

```diff
@@ -26,133 +28,114 @@ hardware) by running `qvm-start sys-usb`.
 5. Update dom0 and install `make`, referring again to
 [the next section of SDW docs](https://workstation.securedrop.org/en/stable/admin/install.html#apply-dom0-updates-estimated-wait-time-15-30-minutes)
 
+In our case, we also install `open-vm-tools` and run `sudo systemctl enable vmtoolsd`,
+as our scripts use vmtoolsd to issue commands to the dom0 from the VMware API.
+
 ```
-sudo qubes-dom0-update make
+sudo qubes-dom0-update make open-vm-tools
 ```
 
 6. Run any updates you see in the Qubes menu and then reboot.
 
-7. In dom0, create the sd-ssh StandaloneVM:
+7. In dom0, create the sd-dev StandaloneVM. If it's Qubes 4.2, you can use the fedora-38-xfce template.
 
 ```
-sudo qvm-create --standalone --template fedora-37 --label red sd-ssh
-qvm-volume resize sd-ssh:root 50G
-qvm-volume resize sd-ssh:private 20G
-qvm-tags sd-ssh add sd-client
+sudo qvm-create --standalone --template fedora-38 --label red sd-dev
+qvm-volume resize sd-dev:root 50G
+qvm-volume resize sd-dev:private 20G
 ```
 
 Also ensure that you check the box to 'Start qube automatically on boot' in the Qubes settings.
 
-# Install dependencies on sd-ssh VM
+# Install dependencies on sd-dev VM
 
-1. Open a terminal in the sd-ssh VM and perform the following steps to install the core dependencies:
+1. Open a terminal in the sd-dev VM and perform the following steps to install the core dependencies:
 
 ```
-sudo dnf install openssh-server rpm-build dnf-plugins-core python3-pip python3-flask python3-paramiko python3-scp
-sudo pip3 install python-dotenv github-webhook
-sudo systemctl enable sshd
-sudo systemctl start sshd
+sudo dnf install rpm-build dnf-plugins-core
 ```
 
-2. Install Tailscale:
+2. Setup docker:
 
 ```
-curl -fsSL https://tailscale.com/install.sh | sh
-sudo tailscale up --advertise-tags=tag:servers,tag:sd-ci-servers
+sudo dnf config-manager --add-repo https://download.docker.com/linux/fedora/docker-ce.repo
```

Comment:
Is this installing docker solely for the purposes of building the securedrop-workstation rpm? We also support podman (personally I prefer it), which should just require `sudo dnf install podman` instead of configuring an external repo.

Reply:
> Is this installing docker solely for the purposes of building the securedrop-workstation rpm?

I think so - was just following instructions when setting up the original CI a year ago.

If podman works, I'll do that. It's not come up before. Thanks